src/Microsoft.AspNet.Mvc.Core/ResponseCacheLocation.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// Determines the value for the "Cache-control" header in the response.
+    /// </summary>
+    public enum ResponseCacheLocation
+    {
+        // Cached in both proxies and client
+        Any = 0,
+        // Cached only in the client
+        Client = 1,
+        // The "Location" part of "Cache-control" is not set
+        Default = 2
```

Comment:
Is the semantics for `Default` different from `None`? If not, let's just call it 1. IMO, we don't have to preserve the values for these enums.